gravitee-apim-console-webui/src/management/application/creation-ng/application-creation-form.stories.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Meta, moduleMetadata, StoryObj } from '@storybook/angular';
+import { of } from 'rxjs';
+
+import { ApplicationCreationComponent } from './application-creation.component';
+
+import { fakeApplicationTypes } from '../../../entities/application-type/ApplicationType.fixture';
+import { ApplicationTypesService } from '../../../services-ngx/application-types.service';
+
+export default {
```

Comment:
Do you think that we could add another story with the environment setting `Enable Dynamic Client Registration` disabled ? It would allow the developers to see the 2 displays possible for this page.

Reply:
🚀